### PR TITLE
feat: enable gpu

### PR DESCRIPTION
**Pull request by azayz**

### CODE REVIEW

File: now/constants.py [hunk 1]
```diff
@@ -5,7 +5,8 @@
 # DEMO_DATASET_DOCARRAY_VERSION = docarray_version
 DEMO_DATASET_DOCARRAY_VERSION = '0.13.17'
 DOCKER_BFF_PLAYGROUND_TAG = '0.0.105'
-NOW_PREPROCESSOR_VERSION = '0.0.35'
+NOW_PREPROCESSOR_VERSION = '0.0.34-enable-gpu'
+GPU_THRESHOLD = 50000
```
Comment (joschkabraun) [inline now/constants.py:9]: as dicussed in pairing would increase this to larger than 200_000 such that demo datasets don't trigger GPU

**Review by joschkabraun — COMMENTED** [review]
Nice PR! After fixing the minor comments, the other flow.yaml and adding a test it LGTM :)

**Review by florian-hoenicke — APPROVED** [review]
LGTM after joschkas feedback is addressed

**Comment by florian-hoenicke** [comment]
I would recommend ignoring the music case for GPU because there is low business value behind currently.

**Comment by florian-hoenicke** [comment]
In our current setup, it is always required to update the preprocessor version with each commit.

**Pull request closed without merging**